recipes/libxsmm/meta.yaml

```diff
@@ -0,0 +1,54 @@
+{% set name = "libxsmm" %}
+{% set version = "1.13" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/hfp/{{ name }}/archive/{{ version }}.tar.gz
+  sha256: 47c034e169820a9633770eece0e0fdd8d4a744e09b81da2af8c2608a4625811e
+
+build:
+  number: 0
+  skip: True  # [win]
+  run_exports:
+    - {{ pin_subpackage('libxsmm', max_pin='x.x') }}
```

Comment:
@isuruf you said we should use `x` here, isn't it?

Reply:
Looks like one `x` is appropriate: https://github.com/hfp/libxsmm/issues/282